app/main/views/g_cloud.py

```diff
@@ -443,8 +446,24 @@ def view_project(framework_framework, project_id):
 def end_search(framework_framework, project_id):
     all_frameworks = data_api_client.find_frameworks().get('frameworks')
     framework = framework_helpers.get_latest_live_framework(all_frameworks, framework_framework)
+    frameworks_by_slug = framework_helpers.get_frameworks_by_slug(data_api_client)
+
+    # Get the requested Direct Award Project.
+    project = data_api_client.get_direct_award_project(project_id=project_id)['project']
+    if not is_direct_award_project_accessible(project, current_user.id):
+        abort(404)
+
+    searches = data_api_client.find_direct_award_project_searches(user_id=current_user.id,
+                                                                  project_id=project['id'])['searches']
+    search = list(filter(lambda x: x['active'], searches))[0]
```

Comment:
We don't need to do this list+filter any more if you pass `active=True` to the `find_direct_award_project_searches` call (assuming you have a recent-ish apiclient on this branch).